packages/zowe-explorer/src/Profiles.ts

```diff
@@ -1237,6 +1237,90 @@ export class Profiles extends ProfilesCache {
         }
     }
 
+    public async basicAuthClearSecureArray(profileName?: string): Promise<void> {
+        const profAttrs = this.getProfileFromConfig(profileName);
+        const configApi = (await this.getProfileInfo()).getTeamConfig();
+        configApi.set(`${(await profAttrs).profLoc.jsonLoc}.secure`, []);
+        configApi.delete(`${(await profAttrs).profLoc.jsonLoc}.properties.user`);
+        configApi.delete(`${(await profAttrs).profLoc.jsonLoc}.properties.password`);
```

Comment:
If you add an `await` on line 1241, you can avoid awaiting each time in lines 1243-1245.